### PR TITLE
EOS-13904: Remove redunent config of STORAGE_ENCLOSURE from consul config (pre-cortx-1.0)

### PR DESCRIPTION
**Pull request by mariyappanp**

### CODE REVIEW

**Review by thavanathan — APPROVED** [review]

File: pillar/components/sspl.sls [hunk 1]
```diff
@@ -96,12 +96,6 @@ sspl:
     threaded: "true"
     logging_interval: "3600"
   STORAGE_ENCLOSURE:
-    primary_controller_ip: "127.0.0.1"
```
Comment (cdeshmukh) [inline pillar/components/sspl.sls:99]: https://github.com/Seagate/cortx-prvsnr/pull/583#discussion_r499727186

same comment applies. So what else is the cleanup candidate here ?

Comment (ypise) [reply pillar/components/sspl.sls:99]: @shubhamseagate 
You might want to remove:
```
  IPMI:
    user: "admin"
    pass: "admin"
  RABBITMQINGRESSPROCESSOR:
    virtual_host: "SSPL"
    queue_name: "actuator-req-queue"
    exchange_name: "sspl-in"
    routing_key: "actuator-req-key"
    username: "sspluser"
    password: "sspl4ever"
    primary_rabbitmq_host: "localhost"
  RABBITMQEGRESSPROCESSOR:
    virtual_host: "SSPL"
    queue_name: "sensor-queue"
    exchange_name: "sspl-out"
    routing_key: "sensor-key"
    ack_queue_name: "sensor-queue"
    ack_exchange_name: "sspl-out"
    ack_routing_key: "sensor-key"
    username: "sspluser"
    password: "sspl4ever"
    message_signature_username: "sspl-ll"
    message_signature_token: "ALOIUD986798df69a8koDISLKJ282983"
    message_signature_expires: "3600"
    iem_route_exchange_name: "sspl-out"
    primary_rabbitmq_host: "localhost"

```

**Review by ypise — CHANGES_REQUESTED** [review]

File: pillar/components/sspl.sls (hunk 1 above)
Comment (cdeshmukh) [reply pillar/components/sspl.sls:99]: @ypise  @mariyappanp https://github.com/Seagate/cortx-prvsnr/pull/592 is taking of this change too. Lets keep one PR only and discard this now

**Comment by mariyappanp** [comment]
Addressing this from @shubhamseagate patch.

**Pull request closed without merging**